static/css/main.css

```diff
@@ -88,7 +85,7 @@ code {
     text-align: center;
     font-size: 25pt;
     font-weight: bold;
-    margin: 10px;
+    margin: 18px 10px 0;
```

Comment:
This may feel like we're replacing one magic number (`95.5vh`) with another (`18px`). This isn't true though, since the `18px` is calculated via the existing `10px` that was there plus an additional `8px`. Those `8px` are part of the default margin's that are applied to every `body` tag, unless explicitly set otherwise. Since we've now zeroed out the margin in the `body` (a `body` with `height: 100%` and a positive `margin` causes the page to scroll ever so slightly), we've had to add this margin (the `8px`) to the sub-components.